src/transformers/pipelines/text_to_audio.py

```diff
@@ -0,0 +1,151 @@
+from typing import List, Union
```

Comment:
Missing the copyright here. Please copy it from another pipeline and switch the year to 2023.